content/pt-br/docs/concepts/overview/working-with-objects/object-management.md

```diff
@@ -0,0 +1,180 @@
+- Os comandos não fornecem uma fonte de registros, exceto o que está ativo.
+- Os comandos não fornecem um *template* para a criação de novos objetos.
+
+## Configuração do objeto imperativo
+
+Na configuração do objeto imperativo o comando kubectl especifica a operação
+(criar, substituir, etc.), *flags* opcionais e pelo menos uma arquivo de nomes. 
+O arquivo especificado deve conter uma definição completa do objeto em formato
+YAML ou JSON. 
+
+Veja a [referência da API ](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/)
+para mais detalhes sobre as definições de objetos.
+
+{{< warning >}}
+O comando imperativo `replace` substitui a especificação exstente
+coom uma nova, descartando todas as alterações do objeto ausente do
+arquivo de configuração. Essa abordagem não deve ser utilizada com 
+tipos de recursos cujas especificações são atualizadas independentemente 
+do arquivo de configuração. Serviços do tipo `LoadBalancer`, por exemplo, 
+possuem seu campo `externalIPs` atualizado independente da configuração 
+do cluster.
+{{< /warning >}}
+
+### Exemplos
+
+Crie os objetos definidos no arquivo de configuração: 
+
+```sh
+kubectl create -f nginx.yaml
+```
+
+Exclua os objetos definidos em dois arquivos de cofiguração:
+
+```sh
+kubectl delete -f nginx.yaml -f redis.yaml
+```
+
+Atualize os objetos definidos em um arquivo de configuração substituindo
+a configuração ativa:
+
+```sh
+kubectl replace -f nginx.yaml
+```
+
+### Comparações 
+
+Vantagens comparadas à comandos imperativos:
+
+- A configuração do objeto pode ser salva em um sistema de controle de código, como o Git.
+- A configuração do objeto pode integrar com processos como revisão de mudanças antes de *push* e trilhas de auditoria.
+- A configuração do objeto fornece um *template* para criar novos objetos.
+
+Desvantagens comparadas aos comandos imperativos:
+
+- A configuração do objeto exige um entendimento básico ao esquema de objetos.
+- A configuração do objeto exige um passo adicional para escrever o arquivo YAML.
+
+Vantagens comparadas à configuração da declaração do objeto:
+
+- O comportamento de configuração do objeto imperativo é simples e fácil de entender.
+- A partir da versão 1.5 do Kubernetes a configuração do objeto imperativo é mais madura.
+
+Desvantagens comparadas à configuração da declaração do objeto:
+
+- A configuração do objeto imperativo funciona melhor em arquivos e não diretórios.
+- Atualizações a objetos ativos devem ser mencionadas nos arquivos de configuração ou serão perdidas durante a próxima substituição.
+
+## Configuração da declaração do objeto
+
+Ao usar configuração da declaração de objeto, um usuário opera em arquivos 
+de configuração de objetos armazenados localmente, entretanto o usuário não
+define as operações a serem realizadas nos arquivos. Operações de criar, atualizar
+e deletar são detectadas automaticamente por objeto pelo `kubectl`. Isso hablita
```

Comment:
Suggestion: using "permite" instead of "habilita" would read better here.

Reply:
@tamarabarbosa ping :)